releng/org.eclipse.n4js.targetplatform/org.eclipse.n4js.targetplatform.target

```diff
@@ -51,15 +52,17 @@
     <location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
       <unit id="ch.qos.logback.classic" version="1.0.7.v20121108-1250"/>
       <unit id="com.fasterxml.jackson.core.jackson-databind" version="2.5.0.v201504151636"/>
+      <unit id="com.google.guava" version="15.0.0.v201403281430"/>
       <unit id="com.google.inject" version="3.0.0.v201605172100"/>
       <repository location="http://download.eclipse.org/tools/orbit/downloads/drops/R20160520211859/repository"/>
     </location>
     <location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
-      <unit id="it.xsemantics.runtime" version="1.10.2.v20170211-0945"/>
-      <repository location="https://dl.bintray.com/lorenzobettini/xsemantics/updates/1.10"/>
+      <unit id="it.xsemantics.runtime" version="1.12.0.v20170627-2011"/>
+      <repository location="http://dl.bintray.com/lorenzobettini/xsemantics/updates/1.12"/>
     </location>
     <location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
-      <unit id="org.xpect.xtext.lib" version="0.1.0.201611181623"/>
+      <unit id="org.eclipse.jdt.junit.runners" version="0.1.0.201411181552"/>
```

Comment:
this seems to be a new dependency, downloaded from an update site from Moritz? Is that ok?